torch/distributions/chi2.py

```diff
@@ -4,8 +4,8 @@
 
 class Chi2(Gamma):
     r"""
-    Creates a Chi2 distribution parameterized by shape parameter :attr:`df`.
-    This is exactly equivalent to ``Gamma(alpha=0.5*df, beta=0.5)``
+    Creates a Chi^2 (Chi-squared) distribution parameterized by shape parameter 
```

Comment:
There's a trailing space here that the linter is complaining about. Could you remove that and push again?